edshipyard.py

```diff
@@ -21,29 +24,44 @@
 
 # Ship masses
 # TODO: prefer something other than pickle for this storage (dev readability, security)
-ships = pickle.load(open(join(config.respath_path, 'ships.p'), 'rb'))
+ships = pickle.load(open(pathlib.Path(config.respath_path) / 'ships.p', 'rb'))
+
+
+def export(data, filename=None) -> None:  # noqa: C901, CCR001
+    """
+    Export ship loadout in E:D Shipyard plain text format.
 
+    :param data: CAPI data.
+    :param filename: Override default file name.
+    """
+    def class_rating(module: __Module) -> str:
+        """
+        Return a string representation of the class of the given module.
 
-# Export ship loadout in E:D Shipyard plain text format
-def export(data, filename=None):
-    def class_rating(module: __Module):
+        :param module: Module data dict.
+        :return: Rating of the module.
+        """
         mod_class = module['class']
         mod_rating = module['rating']
         mod_mount = module.get('mount')
-        mod_guidance = module.get('guidance')
+        mod_guidance: str = module.get('guidance')  # type: ignore
```

Comment:
```suggestion
        mod_guidance: str = str(module.get('guidance'))
```
Just to force it and drop the need for the ignore